dev-python/pygobject/pygobject-3.52.3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_EXT=1
+DISTUTILS_USE_PEP517=no
+GNOME_TARBALL_SUFFIX="gz"
+PYTHON_COMPAT=( python3_{10..13} pypy3 pypy3_11 )
+
+inherit gnome.org meson virtualx xdg distutils-r1
+
+DESCRIPTION="Python bindings for GObject Introspection"
+HOMEPAGE="
+	https://pygobject.gnome.org/
+	https://gitlab.gnome.org/GNOME/pygobject/
+"
+SRC_URI+="
+	https://github.com/python/pythoncapi-compat/archive/0a8b2c56331a31d7f7096faaa1c1c26467b51c15.tar.gz -> \
+		${P}_custom_pythoncapi-compat.gh.tar.gz
+"
+LICENSE="LGPL-2.1+"
+SLOT="3"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
+IUSE="+cairo test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.80:2
+	>=dev-libs/gobject-introspection-1.84:=
+	dev-libs/libffi:=
+	cairo? (
+		>=dev-python/pycairo-1.16.0[${PYTHON_USEDEP}]
+		x11-libs/cairo[glib]
+	)
+"
+DEPEND="
+	${RDEPEND}
+	test? (
+		>=app-accessibility/at-spi2-core-2.46.0[introspection]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		x11-libs/gdk-pixbuf:2[introspection,jpeg]
+		x11-libs/gtk+:3[introspection]
+		x11-libs/pango[introspection]
+	)
+"
+BDEPEND="
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	default
+
+	einfo "Intentionally overwriting subprojects/pythoncapi-compat with a newer revision than" \
+		  "currently set in pygobject-3.52.3/work/pygobject-3.52.3/subprojects/pythoncapi-compat.wrap" \
+		  "to fix build for PYTHON_TARGETS='pypy3_11'"
```

Comment:
It's probably not necessary to `einfo` about this, but if we do then we should line break the message across multiple calls to `einfo`.